sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultAccessControlClientBuilder.java

```diff
@@ -44,35 +44,52 @@
 
 /**
  * This class provides a fluent builder API to help aid the configuration and instantiation of the
- * {@link KeyVaultAccessControlAsyncClient} and {@link KeyVaultAccessControlClient}, by calling
- * {@link KeyVaultAccessControlClientBuilder#buildAsyncClient()} and
- * {@link KeyVaultAccessControlClientBuilder#buildClient()} respectively. It constructs an instance of the desired
- * client.
+ * {@link KeyVaultAccessControlAsyncClient access control async client} and
+ * {@link KeyVaultAccessControlClient access control sync client}, by calling
+ * {@link KeyVaultAccessControlClientBuilder#buildAsyncClient() buildAsyncClient} and
+ * {@link KeyVaultAccessControlClientBuilder#buildClient() buildClient} respectively. It constructs an instance of the
+ * desired client.
  *
- * <p> The minimal configuration options required by {@link KeyVaultAccessControlClientBuilder} to build an
- * an {@link KeyVaultAccessControlAsyncClient} are {@link String vaultUrl} and {@link TokenCredential credential}.</p>
+ * <p> The minimal configuration options required by {@link KeyVaultAccessControlClientBuilder} to build a
+ * {@link KeyVaultAccessControlAsyncClient} are {@link String vaultUrl} and {@link TokenCredential credential}.</p>
  *
- * <p><strong>Samples to construct a sync client</strong></p>
- * <!-- src_embed com.azure.security.keyvault.administration.keyVaultAccessControlClient.instantiation -->
+ * <!-- src_embed com.azure.security.keyvault.administration.KeyVaultAccessControlAsyncClient.instantiation -->
  * <pre>
- * KeyVaultAccessControlClient keyVaultAccessControlClient = new KeyVaultAccessControlClientBuilder&#40;&#41;
+ * KeyVaultAccessControlAsyncClient keyVaultAccessControlAsyncClient = new KeyVaultAccessControlClientBuilder&#40;&#41;
  *     .vaultUrl&#40;&quot;&lt;your-managed-hsm-url&gt;&quot;&#41;
  *     .credential&#40;new DefaultAzureCredentialBuilder&#40;&#41;.build&#40;&#41;&#41;
- *     .buildClient&#40;&#41;;
+ *     .buildAsyncClient&#40;&#41;;
  * </pre>
- * <!-- end com.azure.security.keyvault.administration.keyVaultAccessControlClient.instantiation -->
- * <p><strong>Samples to construct an async client</strong></p>
- * <!-- src_embed com.azure.security.keyvault.administration.keyVaultAccessControlAsyncClient.instantiation -->
+ * <!-- end com.azure.security.keyvault.administration.KeyVaultAccessControlAsyncClient.instantiation -->
+ *
+ * <p>The {@link HttpLogDetailLevel log detail level}, multiple custom {@link HttpLoggingPolicy policies} and custom
```

Comment:
Should state its HTTP log detail level as it doesn't change how other logging is done. We're also setting HttpLogOptions and not just the log level in the sample